packages/instrumenter/src/transformers/directive-bookkeeper.ts

```diff
@@ -89,4 +96,21 @@ export class DirectiveBookkeeper {
     mutatorName = mutatorName.toLowerCase();
     return this.currentIgnoreRule.findIgnoreReason(mutatorName, line);
   }
+
+  private warnAboutUnusedDirective(mutators: string, directiveType: string, scope: string) {
+    const directives = mutators
+      .split(',')
+      .map((mutator) => mutator.trim())
+      .filter((mutator) => mutator !== WILDCARD);
+    for (const directive of directives) {
+      if (!this.allMutators.map((x) => x.name.toLowerCase()).includes(directive.toLowerCase())) {
+        this.logger.warn(
+          // Scope can be global and therefore undefined
+          `Unused 'Stryker ${
+            scope ? directiveType + ' ' + scope : directiveType
+          }' directive. Mutator with name '${directive}' not found. Directive found at: ${this.originFileName}.`
```

Comment:
We can use `comment.loc!.start.line` and `comment.loc!.start.column` to also print the location. That would be awesome!

Reply:
I probably should have placed a comment for this, but I'll quote the PR description

> I've not included the line number and column since these did not correctly seem to tell where the directive was found, and in case of a next-line directive it had to subtract 1. I think the filename should be sufficient though to find the unused directive. WDYT?

Reply:
I'll retest and implement the feedback somewhere tomorrow or so.  Thanks!

Reply:
Seems to work fine :) 

![image](https://user-images.githubusercontent.com/1828233/201467982-976275f1-aae3-402b-9f3b-b800a86a9018.png)

Reply:
Ah there are comment nodes as well, I was looking at the nodes where the mutation is applied. Thanks!